scripts/qa_crowbarsetup.sh

```diff
@@ -3020,7 +3020,13 @@ function oncontroller_nova_evacuate
     addfloatingip $vm_name
     local floatingip=$(openstack server show -c addresses --format value $vm_name | cut -d " " -f 2)
     # Update security group for icmp
-    nova secgroup-add-rule default icmp -1 -1 0.0.0.0/0 &> /dev/null
+    if iscloudver 8plus; then
+        local projectid=$(openstack project show -c id --format value $OS_PROJECT_NAME)
+        local secgroupid=$(openstack security group list -c ID -c Project --format value | grep $projectid | cut -d " " -f 1)
```

Comment:
Could that be: openstack security group list --project $projectid -c ID --format value instead of using grep and cut?

Reply:
@toabctl I wish it were so, then this will be a lot simpler. Unfortunately, there is no --project option for 'security group list', nor is there any other way of restricting the output to a single project.

Reply:
There is:
```
root@d52-54-77-77-01-01:~ # openstack security group list -h|grep project
                                     [--project <project>]
                                     [--project-domain <project-domain>]
  --project <project>   List security groups according to the project (name or
  --project-domain <project-domain>
                        Domain the project belongs to (name or ID). This can
                        be used in case collisions between project names
```

Reply:
I'm sorry, you're right, there is. I was actually thinking of the `openstack security group rule ... --project` option, that has no effect, no matter how you use it, which is why I had to deal with this:
```
openstack security group rule create --protocol icmp --prefix 0.0.0.0/0 --project openstack default
More than one SecurityGroup exists with the name 'default'.
```